packages/auth/src/AuthProvider/AuthProviderState.ts

```diff
@@ -1,6 +1,7 @@
 import { CurrentUser } from '../AuthContext'
 
 export type AuthProviderState<TUser> = {
+  client?: any
```

Comment:
@Tobbe what do you think the type should be here?

Reply:
TClient

Reply:
See the fix here: https://github.com/redwoodjs/redwood/pull/6897/commits/cc2f430874acb20550ce8b37635e97475ef3deea

I copied it from `AuthImplementation` https://github.com/redwoodjs/redwood/blob/main/packages/auth/src/AuthImplementation.ts#L10

Reply:
That's perfect 👍